versions.tf

```diff
@@ -18,12 +18,10 @@ terraform {
   required_version = ">=0.13.0"
   required_providers {
     google = {
-      source  = "hashicorp/google"
-      version = "~> 4.28"
```

Comment:
Since this is a module versions let's keep the <6.0?

Reply:
Yup, good catch, probably a bad find criteria.